waspc/data/Generator/templates/react-app/src/index.tsx

```diff
@@ -7,7 +7,7 @@ import router from './router'
 import {
   initializeQueryClient,
   queryClientInitialized,
-} from 'wasp/client/operations'
+} from 'wasp/client/operations/queryClient'
```

Comment:
All SDK imports in the framework code have been updated according to the new code organization.